Package request: radeontop

https://github.com/clbr/radeontop

So I wrote this

# Template file for 'radeontop'
pkgname=radeontop
version=v0.8
revision=1
wrksrc="${pkgname}-${version#?}"
build_style=gnu-makefile
hostmakedepends="pkg-config git gettext"
makedepends="ncurses-devel"
depends="xf86-video-ati"
short_desc="View radeon GPU utilization"
maintainer="Logen K <logen@sudotask.com>"
license="GPL-3"
homepage="https://github.com/clbr/radeontop"
distfiles="https://github.com/clbr/radeontop/archive/${version}.tar.gz"
checksum=4430e1aaca895d70ea496a9b14010d9d94cadd6269b28a66104af89ff29d8489

But it has an error:

=> radeontop-v0.8_1: running do_build ...
./getver.sh
fatal: Not a git repository (or any of the parent directories): .git
cc -mtune=generic -O2 -pipe -fstack-protector-strong -D_FORTIFY_SOURCE=2         -c -o detect.o detect.c
detect.c:17:23: fatal error: radeontop.h: No such file or directory=
 #include "radeontop.h"
                       ^
compilation terminated.
<builtin>: recipe for target 'detect.o' failed
make: *** [detect.o] Error 1

I know that it works fine with make via the command line, I also know that radeon.h is in the program's /include folder.  I'm not sure how to let xbps know where that file is.

--- a/srcpkgs/radeontop/template
+++ b/srcpkgs/radeontop/template
@@ -3,8 +3,7 @@ pkgname=radeontop
 version=v0.8
 revision=1
 wrksrc="${pkgname}-${version#?}"
-build_style=gnu-makefile
-hostmakedepends="pkg-config git gettext"
+hostmakedepends="pkg-config"
 makedepends="ncurses-devel libdrm-devel"
 depends="xf86-video-ati"
 short_desc="View radeon GPU utilization"
@@ -15,7 +14,9 @@ distfiles="https://github.com/clbr/radeontop/archive/${version}.tar.gz"
 checksum=4430e1aaca895d70ea496a9b14010d9d94cadd6269b28a66104af89ff29d8489
 
 do_build() {
-	make CC="$CC" CXX="$CXX" LD="$LD" AR="$AR" RANLIB="$RANLIB" \
-			CPP="$CPP" AS="$AS" OBJDUMP="$OBJDUMP" STRIP="true"
+	make CC="$CC"
 }
 
+do_install() {
+	vbin radeontop
+}

--- a/srcpkgs/radeontop/template
+++ b/srcpkgs/radeontop/template
@@ -1,0 +1,21 @@
+# Template file for 'radeontop'
+pkgname=radeontop
+version=v0.8
+revision=1
+wrksrc="${pkgname}-${version#?}"
+build_style=gnu-makefile
+hostmakedepends="pkg-config git gettext"
+makedepends="ncurses-devel libdrm-devel"
+depends="xf86-video-ati"
+short_desc="View radeon GPU utilization"
+maintainer="Logen K <logen@sudotask.com>"
+license="GPL-3"
+homepage="https://github.com/clbr/radeontop"
+distfiles="https://github.com/clbr/radeontop/archive/${version}.tar.gz"
+checksum=4430e1aaca895d70ea496a9b14010d9d94cadd6269b28a66104af89ff29d8489
+
+do_build() {
+	make CC="$CC" CXX="$CXX" LD="$LD" AR="$AR" RANLIB="$RANLIB" \
+			CPP="$CPP" AS="$AS" OBJDUMP="$OBJDUMP" STRIP="true"
+}
+

